app/src/main/java/org/dosomething/letsdothis/network/models/LoginResponse.java

```diff
@@ -0,0 +1,21 @@
+package org.dosomething.letsdothis.network.models;
+/**
+ * Created by toidiu on 4/16/15.
+ */
+public class LoginResponse
+{
+    public String email;
+    public String phone;
+    public String created_at;
```

Comment:
eventually yes but right now i have no need for the date object(i guess when we update..) so i want to code that up when i do that.